extensions/amp-sticky-ad/OWNERS

```diff
@@ -4,7 +4,7 @@
 {
   rules: [
     {
-      owners: [{name: 'zhouyx'}],
+      owners: [{name: 'ampproject/wg-ads-reviewers'}],
```

Comment:
I would like to have myself added as an owner here please.

Reply:
done

Reply:
Thanks